sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/compatibility/SpringCloudAzureCompatibilityVerifierProperties.java

```diff
@@ -12,24 +13,44 @@
  */
 @ConfigurationProperties("spring.cloud.azure.compatibility-verifier")
 public class SpringCloudAzureCompatibilityVerifierProperties {
+
+    /**
+     * Whether to enable the Spring Cloud Azure compatibility verifier.
+     */
     private boolean enabled;
+    /**
+     * Comma-delimited list of compatible Spring Boot versions.
```

Comment:
comment may not correctly describe the field.

Reply:
Do you have suggestions for this?